core/tests/protractor_utils/TopicAndStoryViewerPage.js

```diff
@@ -55,11 +53,6 @@ var TopicAndStoryViewerPage = function() {
       lessonTrack, 'Lesson track takes too long to be visible.');
     expect(await lessonUncompletedIcons.count()).toEqual(count);
   };
-
-  this.login = async function(email, username) {
-    await action.click('Login button', loginButton);
-    await users.completeLoginFlowFromStoryViewerPage(email, username);
```

Comment:
I'm seeing errors in TopicAndStoryViewerPage e2e tests. Maybe they result from the way this was changed?

Should topicAndStoryViewer.js do something equivalent to completeLoginFlowFromStoryViewerPage() manually?

Reply:
@brianrodri Can you please reply to this comment with an explanation of what actually happened?

Reply:
This function and `completeLoginFlowFromStoryViewerPage` did not do anything special enough to deserve their own function.  I've removed it in favor of the `createUserAndLogin` function instead.